CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1473,7 +1502,7 @@ private void RemoveSourceHook()
         /// <returns>bool to indicate if browser was created. If the browser has already been created then this will return false.</returns>
         protected virtual bool CreateOffscreenBrowser(Size size)
         {
-            if (browserCreated || System.ComponentModel.DesignerProperties.GetIsInDesignMode(this) || size.IsEmpty || size.Equals(new Size(0, 0)))
+            if (browserCreated || size.IsEmpty || size.Equals(new Size(0, 0)))
```

Comment:
`CreateOffscreenBrowser` gets called from `OnActualSizeChanged` which isn't registered when in design mode.